documentation/publish/distribute-sideloading.md

```diff
@@ -0,0 +1,256 @@
+---
+layout: sidebar
+title: Add-ons for desktop apps
+permalink: /documentation/publish/distribute-sideloading/
+published: false
+topic: Publish
+tags: [add-on, distribution, sideloading, guide, installation]
+contributors: [irenesmith, jwilk, hellosct1, gray_-_wolf, luanmm, rebloor]
+last_updated_by: hellosct1
+date: 2019-03-18 05:05:42
+---
+
+<!-- Page Hero Banner -->
+
+{% capture page_hero_banner_content %}
+
+# Sideloading add-ons
+
+You may want to send a user your add-on’s XPI file by some means other than a web download, such as an email distribution of a beta version for user testing. In this case, there are two practical options for installing the add-on:
+
+- Using **Install Add-on From File** in the Add-ons Manager.
+- Adding the file to one of the standard extension folders.
+
+<!-- Note -->
+
+{% capture note %}
+
+No automatic updates will be performed for add-ons installed using these methods. You will have to deliver a new XPI file to your user for each update. However, automatic compatibility checks are still performed.
+
+{% endcapture %}
+{% include modules/note.html
+	content=note
+	alert=false
+%}
+
+<!-- END: Note -->
+
+{% endcapture %}
+{% include modules/page-hero.html
+	content=page_hero_banner_content
+%}
+
+<!-- END: Page Hero Banner -->
+
+<!-- Content with Table of Contents Module -->
+
+{% capture content_with_toc %}
+
+## Preparing your add-on
+
+Regardless of the sideloading method used, you must prepare the add-on as follows:
+
+1. Ensure the add-on includes an ID, by adding the following to its manifest.json file, replacing **_your-add-on-name@your-domain.com_** with a suitable ID for your add-on:
+
+<!-- Syntax Highlighting -->
+
+{% highlight javascript linenos %}
+"applications": {
+"gecko": {
+"id": "your-add-on-name@your-domain.com"
+}
+}
+{% endhighlight %}
+
+<!-- END: Syntax Highlighting -->
+
+An email address style ID is recommended.
+
+{:start="2"}
+
+2. Sign the add-on in [addons.mozilla.org](https://addons.mozilla.org) (AMO). Depending on how you want to make your add-on available, you can use either the unlisted (if you are distributing the add-on exclusively) or listed options. For more details, see [Signing and distributing your add-on](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/Distribution).
```

Comment:
no fix needed for now, just noting "Signing and distribution your add-on" should link back to EW once that page is up.